integration-cli/docker_cli_rm_test.go

```diff
@@ -9,34 +10,50 @@ import (
 )
 
 func (s *DockerSuite) TestRmContainerWithRemovedVolume(c *check.C) {
-	testRequires(c, DaemonIsLinux)
 	testRequires(c, SameHostDaemon)
 
-	dockerCmd(c, "run", "--name", "losemyvolumes", "-v", "/tmp/testing:/test", "busybox", "true")
+	prefix := ""
```

Comment:
@jhowardmsft we're doing this in multiple place, could we add a*util* function that returns `prefix` and `slash` according to the specified `daemonPlatform` (and thus making it easier to change, fix, enhance, maintain) ? :angel: